utils/compiler/create_snapshot_entry.dart

```diff
@@ -10,7 +10,7 @@ import 'dart:io';
 import 'dart:async';
 
 Future<String> getVersion(var rootPath) {
-  var suffix = Platform.operatingSystem == 'windows' ? '.exe' : '';
+  var suffix = Platform.operatingSystem == 'windows' ? '.bat' : '';
   var printVersionScript = rootPath.resolve("tools/make_version.py");
   return Process.run(
           "python$suffix", [printVersionScript.toFilePath(), "--quiet"])
```

Comment:
@johnniwinther sent me the following suggestion:

> I'd add `runInShell: true` here instead and leave out the '$suffix' part. That'll let Windows find the python version that it normally finds. On my setup I actually get different versions depending on the suffix and my default is the .exe version.

Would you mind making that change?